docs/src/pages/demos/snackbars/ConsecutiveSnackbars.tsx

```diff
@@ -74,7 +70,7 @@ class ConsecutiveSnackbars extends React.Component<Props, State> {
 
   render() {
     const { classes } = this.props;
-    const { messageInfo = {} as SnackbarMessage } = this.state;
+    const { messageInfo = {} as Partial<SnackbarMessage> } = this.state;
```

Comment:
This was an unsafe cast. It allowed `messageInfo.message.length` while it might throw at runtime if the queue is empty.